packages/experience-builder-sdk/src/blocks/editor/VisualEditorBlock.tsx

```diff
@@ -130,6 +130,18 @@ export const VisualEditorBlock = ({
             ...acc,
             [variableName]: transformContentValue(value, variableDefinition),
           };
+        } else if (variableMapping.type === 'ComponentValue') {
+          // For design component, we are only handling binding for UnboundValues for now
+          const value = getUnboundValues({
+            key: variableMapping.key,
+            fallback: variableDefinition.defaultValue,
```

Comment:
Aha! This is probably where the [object Object] I talked about [here](https://github.com/contentful/user_interface/pull/18355#pullrequestreview-1743970513) is coming from, coz the default value is an object of type and key/path.

Would be good if we grab the default value from the design component unbound values (coz that's where the actual default values are stored yeah?).